huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_node_pool_test.go

```diff
@@ -193,10 +193,14 @@ func TestAccCCENodePool_podSecurityGroups(t *testing.T) {
 		CheckDestroy:      testAccCheckCCENodePoolDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccCCENodePool_podSecurityGroups(rName),
+				Config: testAccCCENodePool_SecurityGroups(rName),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckCCENodePoolExists(resourceName, clusterName, &nodePool),
 					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttrPair(resourceName, "security_groups.0",
```

Comment:
it's better to check for `pod_security_groups`

Reply:
pod_security_groups not returned by API, so checking it is meaningless